src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -166,11 +165,11 @@ async def get_input_volts_source(self):
         :rtype: int
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_INPUTVOLTSSRC, [MGMSG.PZ_GET_INPUTVOLTSSRC], 1
+            MGMSG.PZ_REQ_INPUTVOLTSSRC, [MGMSG.PZ_GET_INPUTVOLTSSRC], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_INPUTVOLTSSRC, [MGMSG.PZ_GET_INPUTVOLTSSRC], Tpz._CHANNEL
```
Use class constants.